src/layouts/components/Homepage/InfocolsBody.tsx

```diff
@@ -0,0 +1,209 @@
+
+export const InfocolsSectionBody = ({
+  title,
+  subtitle,
+  url,
+  linktext,
+  infoboxes = [],
+  index,
+  errors,
+  infoboxErrors = [],
+}: InfocolsSectionProps) => {
+  const { onChange, onDelete, onDragEnd, onCreate } = useEditableContext()
+
+  return (
+    <Editable.Section>
+      {/* Subtitle */}
+      <FormControl isInvalid={!!errors.subtitle}>
+        <FormLabel>Subtitle</FormLabel>
+        <Input
+          placeholder="This subtitle appears above the title"
+          id={`section-${index}-infocols-subtitle`}
+          value={subtitle}
+          onChange={onChange}
+        />
+        <FormErrorMessage>{errors.subtitle}</FormErrorMessage>
+      </FormControl>
+
+      {/* Title */}
+      <FormControl isRequired isInvalid={!!errors.title}>
+        <FormLabel>Title</FormLabel>
+        <Input
+          placeholder="Your title goes here"
+          id={`section-${index}-infocols-title`}
+          value={title}
+          onChange={onChange}
+        />
+        <FormErrorMessage>{errors.title}</FormErrorMessage>
+      </FormControl>
+
+      {/* Link Text */}
+      <FormControl isInvalid={!!errors.linktext}>
+        <FormLabel>Link text</FormLabel>
+        <Input
+          placeholder="This is shown at the bottom of the section"
+          id={`section-${index}-infocols-linktext`}
+          value={linktext}
+          onChange={onChange}
+        />
+        <FormErrorMessage>{errors.linktext}</FormErrorMessage>
+      </FormControl>
+
+      {/* URL */}
+      <FormControl isInvalid={!!errors.url}>
+        <FormLabel>Link URL</FormLabel>
+        <Input
+          placeholder="Insert /page-url or https://"
+          id={`section-${index}-infocols-url`}
+          value={url}
+          onChange={onChange}
+        />
+        <FormErrorMessage>{errors.url}</FormErrorMessage>
+      </FormControl>
+
+      <Divider py="0.5rem" />
+      <Text mb="0.5rem" textStyle="h5">
+        Infoboxes
+      </Text>
+      <Text textStyle="body-2" mb="1.5rem">
+        You can add up to 4 infoboxes
+      </Text>
+      <DragDropContext onDragEnd={onDragEnd}>
+        <Editable.Droppable width="100%" editableId={`infocolInfobox-${index}`}>
+          <Editable.EmptySection
+            title="Add an infobox to get started"
+            subtitle="You must add at least 1 infobox to this block"
+            isEmpty={infoboxes.length === 0}
+          >
+            <Editable.Accordion>
+              <VStack p={0} spacing="0.75rem">
+                {infoboxes.map((infobox, infoboxIndex) => (
+                  <Editable.DraggableAccordionItem
+                    draggableId={`infocolInfobox-${index}-${infoboxIndex}-draggable`}
+                    index={infoboxIndex}
+                    title={infobox.title}
+                    isInvalid={_.some(infoboxErrors[infoboxIndex])}
+                    isNested
+                  >
+                    <Editable.Section>
+                      {/* infobox Title */}
```

Comment:
infocol*

Reply:
is supposed to be infobox here. infocol is the outer section whereas the infobox is the inner section